### PR TITLE
Add loop over different cuts in ellipses

### PR DESCRIPTION
**Pull request by alesaggio**
Also, add mll_cut_and_inverted_met_cut and WGamma backgrounds

### CODE REVIEW

File: factories_ZA/sampleList.py [hunk 1]
```diff
@@ -48,6 +50,12 @@
     'ggZH_HToBB_ZToNuNu_M125_13TeV_powheg_pythia8_Summer16MiniAODv2'
 ]
 
+#WGamma
+samples_dict["WGamma"] = [
+    'WGToLNuG_TuneCUETP8M1_13TeV-madgraphMLM-pythia8_Summer16MiniAODv2',
+    'WGToLNuG_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8_extended_ext0_plus_ext1_plus_ext2_plus_ext3'
```
Comment (OlivierBondu) [inline factories_ZA/sampleList.py]: `ext0` is actually the sample without the extension, so it means it is duplicated here!

**Review by OlivierBondu — CHANGES_REQUESTED** [review]
Looks good! but one WGamma sample is duplicated :smile:

**Comment by alesaggio** [comment]
Implemented ;)

**Review by OlivierBondu — APPROVED** [review]

**Comment by OlivierBondu** [comment]
Thanks!

**Pull request merged**